executor/agents/base.py

```
@@ -145,6 +162,148 @@ def report_progress(
+
+        self._content_offset += len(content)
+
+        try:
+            send_content_chunk_callback(
+                task_id=self.task_id,
+                subtask_id=self.subtask_id,
+                content=content,
+                offset=self._content_offset,
+                task_type=self.task_type,
+            )
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)